lib/miquido_lints.yaml

```diff
@@ -0,0 +1,336 @@
+          - test/**
+    # - format-comment - disabled as too strict, and comments are - you know - comments
+    - member-ordering-extended: #This is order that works perfect with Flutter Stylizer for VS Code: https://marketplace.visualstudio.com/items?itemName=gmlewis-vscode.flutter-stylizer
+        alphabetize: false
+        order:
+          - constructors
+          - static-public-fields
+          - static-private-fields
+          - fields
+          - overriden-methods
+          - methods
+    # - newline-before-return: - this is just stupid, especially when you have one-line methods
+    #     exclude:
+    #       - test/**
+    - no-boolean-literal-compare
+    - no-empty-block:
+        exclude:
+          - test/**
+    # - no-equal-arguments - disabled as triggered when setting same paddings, sizes in widgets etc...
+    - no-equal-then-else
+    # - no-magic-number - disabled as it makes you extract each height, width (and similar properties) value to separate variables 
+    - no-object-declaration
+    - prefer-async-await
+    # - prefer-conditional-expressions - disabled as sometimes this condition is not as readible as you would expected
+    - prefer-correct-identifier-length:
+        exceptions: ['i', 'x', 'id', 'db']
+        min-identifier-length: 3
+        max-identifier-length: 35
+    - prefer-correct-type-name
+    # - prefer-first - disabled as oldschool array[0] is clearer
+    - prefer-last # but this one is much better that array[array.length-1]
+    - prefer-match-file-name:
+        severity: error
+        exclude:
+          - test/**
+    # - prefer-trailing-comma - this one is pure hardcore and too strict
+    - tag-name:
+        exclude:
+          - test/**
+    - always-remove-listener
+    # - avoid-border-all - proposed solution is adding extra code complexiticy for meaningless profit
+    - avoid-returning-widgets:
+        severity: error
+    - avoid-unnecessary-setstate
+    - avoid-wrapping-in-padding
+    # - prefer-const-border-radius - proposed solution is adding extra code complexiticy for meaningless profit
+    - prefer-extracting-callbacks:
+        ignored-named-arguments:
+          - listener
+          - listenWhen
+          - buildWhen
+    - prefer-single-widget-per-file:
+        ignore-private-widgets: true
+    - prefer-intl-name
+    - provide-correct-intl-args
+
+  metrics:
+    cyclomatic-complexity: 10
+    halstead-volume: 150
+    lines-of-code: 70
+    maintainability-index: 60
+    maximum-nesting-level: 4
+    number-of-parameters: 4
+    source-lines-of-code: 40
+    number-of-methods: 7
+    weight-of-class: 0.33
+
+    technical-debt:
+      threshold: 1
+      todo-cost: 3
+      ignore-cost: 2
+      ignore-for-file-cost: 4
+      as-dynamic-cost: 1
+      deprecated-annotations-cost: 4
+      file-nullsafety-migration-cost: 5
+      unit-type: "MD"
+
+  anti-patterns:
+    - long-method:
+        severity: warning
+        exclude:
+          - test/**
```

Comment:
I know that in concrete projects, developers can use miquido_lints, but choose to ignore some of the lints.
Is the same possible with Dart Code Metrics?

Reply:
yes. by using:
```yaml
dart_code_metrics:
  rules-exclude:
    - ... # configures the list of files that should be ignored by rules
```

although, I think they shouldnt do it. all or nothing!

Reply:
A dodałbyś do Readme w takim razie coś typu: 

**Excluding**
If for some reason (hopefully valid), your team would like to turn off individual rules, then you can do it by:...

Reply:
added excluding section